src/coreclr/System.Private.CoreLib/src/System/Buffer.CoreCLR.cs

```diff
@@ -84,7 +84,7 @@ private static void _BulkMoveWithWriteBarrier(ref byte destination, ref byte sou
         internal static unsafe void Memcpy(byte* dest, byte* src, int len)
         {
             Debug.Assert(len >= 0, "Negative length in memcpy!");
-            Memmove(ref *dest, ref *src, (nuint)len);
+            Memmove(ref *dest, ref *src, (uint)len);
```

Comment:
I wonder if in cases like this it'd be better to do the nuint cast explicitly:
```C#
Memmove(ref *dest, ref *src, (nuint)(uint)len);
```
to help convey to a future reader "yes, this was on purpose, be careful before you fix this 'mistake'".

Reply:
How about `(uint)len /* force zero-extension */`? A little more verbose, but perhaps clearer?

Reply:
That sounds fine,  too. Thanks.